config.js

```diff
@@ -269,6 +269,9 @@ var config = {
     // Enable lock room for all moderators, even when userRolesBasedOnToken is enabled and participants are guests.
     // lockRoomGuestEnabled: false,
 
+    // When enabled the password used for locking a room is restricted to up to 10 digits
```

Comment:
why 10? why not make it configurable, since we are here?

Reply:
Ok I will change it.